ATOMS.md

```diff
@@ -0,0 +1,63 @@
+## Mobiledoc Atoms
+
+Atoms are effectively read-only inline cards.
+
+An atom is an object with a name and at least one of several hooks defined. For example:
+
+```js
+var demoAtom = {
+  name: 'mention',
+  display: {
+    setup(element, options, env, text, payload) {
+      element.innerHTML = `<span class="mention">${text}</span>`;
+    }
+  }
+}
+```
+
+### Available hooks
+
+Like [CARDS](CARDS.md) there are several hooks an atom should define.
+
+```js
+var exampleAtom = {
+  name: 'example',
+  display: {
+    setup(element, options, env, text, payload) {},
+    teardown(setupReturnValue) {}
+  },
+  html: {
+    setup(buffer, options, env, text, payload) {},
+    teardown(setupReturnValue) {}
+  }
+};
+```
+
+|Hook|Used by Mobiledoc Kit|Used by DOM Renderer|Used by HTML Renderer|
+|---|---|---|---|
+|`display`|✓|✓||
+|`html`|||✓|
+
+Atoms are read-only so, unlike cards, do not have an `edit` hook.
+
+Each hook has a `setup` and `teardown` method. The arguments are:
+
+* `element` is a DOM element for that section. Nodes of that view for a atom
+  should be appended to the `element`.
+* `buffer` is an array passed to the `html` hook instead of a DOM element.
+  The content for the atom should be pushed on that array as a string.
+* `options` is the `cardOptions` argument passed to the editor or renderer.
+* `env` contains information about the running of this hook. It may contain
```

Comment:
I took out all the other ones from cards because save/cancel/edit/section didn't seem to be needed for atoms at first glance. 

I can picture a use-case for `remove` such as showing an `x` on hover and letting you click it to remove the atom, but certainly happy to leave that out initially!

If we do have remove, I can think of some use-cases where an atom could be expose an interface which alters it and therefor would want to update its payload. Not something I personally need right now hence why it didn't spring to mind initially.